src/qtt/measurements/new/signal_processor_runner.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from qilib.data_set import DataSet
+
+from qtt.measurements.new.interfaces.signal_processor_interface import SignalProcessorInterface
+
+
+class SignalProcessorRunner:
+    """ A class for post-processing measurement data. """
+
+    def __init__(self) -> None:
+        self._signal_processors: List[SignalProcessorInterface] = []
+
+    def add_signal_processor(self, signal_processor: SignalProcessorInterface) -> None:
+        """ Add a signal processor.
+
+            Args:
+                signal_processor: A signal processor
+        """
+
+        self._signal_processors.append(signal_processor)
```

Comment:
Repeating an earlier comment: Do we need a run-time check that what is passed in is indeed an implementation of SignalProcessorInterface and raise a TypeError otherwise?

Reply:
Done